### ISSUE TITLE
Analytics on the site

### ISSUE THREAD

**Issue by huayra** [varnishcache/homepage#8]
As per 4450349 I wanted to get Google Analytics back to the site, as we have had this since 2010. As per cc2a59a we need to figure this out.

Anyone that is involved in webmastering the site should have access. So all members of the @varnishcache/homepage team have now Manager access to the Google Analytics. 

Re-merging the script shortly.

**Comment by nigoroll** [comment]
I don't want to open another pandoras box, but from the varnishcache project viewpoint, do we _really_ want to tell google all our users do, do we really want to join the global total surveillance frenzy?
I personally blacklist all GA requests in my browser and GA is one of the things which, IMHO, makes google evil.

**Comment by huayra** [comment]
I see your point and I think that as a personal matter is relevant, but I see a point we are overlooking: Do you do that for your customers as well? Is it viable to run a site for aproject runing millions of sites on the web and expect to make improvements in its content and customer journey flow without analytics?

Unfortunately, I do not believe that to be the case.

AWstats is a log analyzer and does not give us all the information we would need over time. So that, an similar software, are not an option.

Piwik is a real option. The only downside would be that it is yet another PHP application to host and maintain. We dropped Drupal so we avoid doing exactly that.

**Comment by nigoroll** [comment]
Yes, I question the use of tracking service also with our clients.
But the varnish-cache project is not a customer and the main question I have is: Do we actually need and use the tracking information (and what for)?
Just because almost everyone uses it doesn't mean it's a good idea and I don't agree that we need to 
> make improvements in [...] customer journey flow
on varnish-cache.org

**Comment by fgsch** [comment]
Tracking != statistics. Are we not happy about the former or the latter?
If it's the former there are other options. As for the latter...

**Comment by huayra** [comment]
So you are proposing that we have no overview of any kind for the traffic we have to our site because your personal opinion is that "google is evil" and that you do not wish us to "join the global total surveillance frenzy"? Or because you see any kind of analytics in general as irrelevant?

I kindly ask you to put a webmaster hat on and think about this in that perspective before replying.

**Comment by bsdphk** [comment]
I have enabled the analytics for now, along with a declaration of our privacy on the front page.

I'm not convinced that the intelligence we derive from analytics is worth the privacy invasion in the long run, but we can discuss that later.

**Comment by bsdphk** [comment]
"Is it viable to run a site for a project running millions of sites on the web and expect to make improvements in its content and customer journey flow without analytics?"

I just want to give my perspective on this bit.

I am sure that by now there is a big industry around SEO-optimized, revenue-generating websites.

That is not us.

I am 100% convinced that the absolute easiest way to improve our website, and by far the way our users will be most happy about, is improving the *content*, not the color, not the animation of the links, but the actual content about what varnish is, how you use varnish, and how to solve your varnish problems.

I looked briefly at what Google Analytics can tell us, and it said precisely the same thing with a lot of fancy graphics and other flim-flam.

I will spend a little bit more time with Google Analytics at some point, and if it doesn't tell me something I don't know, it will be dropped.

As for statistics, I have set up a rolling 7 day varnishlog archive, which we can trawl for statistics.
